lib/OpenQA/Utils.pm

```diff
@@ -558,7 +558,7 @@ sub send_job_to_worker {
     my $res;
     # ugly work around for Net::DBus::Test not being able to handle us using low level API
     return if ref($ipc->{bus}->get_connection) eq 'Net::DBus::Test::MockConnection';
-    eval { $res = $ipc->websockets('ws_send_job', $job); };
+    $res = $ipc->websockets('ws_send_job', $job);
```

Comment:
Does this makes sense? When there is no eval, how would $@ be filled?

Reply:
Oh, I suppose this is one of these magic Perl variables. I don't know. What is `$@` supposed to be anyways? Better give that variable a meaningful name.

At least the `log_debug` line can be removed now, because the logging is now done in `$ipc->websockets` itself.

Reply:
After eval it contains the last exception or empty on no error. If you don't call eval $@ is not overwritten so like ... undefined

Reply:
Only call to `send_job_to_worker` is in `ws_send` and its result is only used one time and only if it is a hash. So for the error case, the assignment of `$res` is never used (if I didn't miss anything).

Since also the debug log is now done in `$ipc->websockets` (purpose of this PR) I removed the useless if-block.

Reply:
in hope that this snipped sheds some light on `eval/$@` relationship: https://github.com/mudler/perl_training/blob/master/examples/eval.pl